### PR TITLE
Vgp233/xile chen/player movement

### PR DESCRIPTION
**Pull request by ScottXileChen**
Added first character "Maria".
Added PlayerBase and Maria class.
Added Maria animations.
Added MariaAC class to control Maria animations.
Baked NavMesh on GameMap scene.

### CODE REVIEW

**Comment by memogtt** [comment]
I think you need to add a reviewer so Erick can receive the request

File: WarSpecialists/Assets/Scenes/GameplayMap_LOCAL_14768.meta [hunk 1]
```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: c96b3e6ee41c2fc4dba30560d84b84ef
+folderAsset: yes
+DefaultImporter:
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant:
```
Comment (erickbordallo) [inline WarSpecialists/Assets/Scenes/GameplayMap_LOCAL_14768.meta]: Remove this Scene. Keep just GamePlay scene. The one with the final map.

File: WarSpecialists/Assets/Scenes/GameplayMap_LOCAL_14768.unity [hunk 2]
```diff
@@ -2552,8 +2542,8 @@ GameObject:
   m_Name: Special_A_Right
   m_TagString: Untagged
   m_Icon: {fileID: 0}
-  m_NavMeshLayer: 0
-  m_StaticEditorFlags: 0
+  m_NavMeshLayer: 1
+  m_StaticEditorFlags: 8
   m_IsActive: 1
 --- !u!4 &2112183478
 Transform:
```
Comment (erickbordallo) [inline WarSpecialists/Assets/Scenes/GameplayMap_LOCAL_14768.unity]: Same. Remove this extra scenes.

**Review by erickbordallo — CHANGES_REQUESTED** [review]
1) Remove that extra scenes. Just use Gameplay scene. The one that contains the current map.
2) Fix the conflicts. Merge master and check the conflicts that you are having. Fix them.

**Comment by ScottXileChen** [comment]
Delete extra scene, and resolve conflict.

**Review by erickbordallo — APPROVED** [review]
Awesome. You can merge your changes.

**Pull request merged**